security-analysis/security-analysis-api/src/main/java/com/powsybl/security/distributed/SecurityAnalysisCommandOptions.java

```diff
@@ -0,0 +1,205 @@
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions parametersFile(Path parametersFile) {
+        this.parametersFile = requireNonNull(parametersFile);
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions taskCount(int taskCount) {
+        this.taskCount = taskCount;
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions outputFile(Function<Integer, Path> outputFile, String format) {
+        this.outputFile = requireNonNull(outputFile);
+        this.outputFileFormat = requireNonNull(format);
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions logFile(Path logFile) {
+        requireNonNull(logFile);
+        this.logFile = i -> logFile;
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions logFile(Function<Integer, Path> logFile) {
+        this.logFile = requireNonNull(logFile);
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions outputFile(Path outputFile, String format) {
+        requireNonNull(outputFile);
+        this.outputFile = i -> outputFile;
+        this.outputFileFormat = requireNonNull(format);
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions taskBasedOnIndex(int taskCount) {
+        return task(i -> new Partition(i + 1, taskCount));
+    }
+
+    public SecurityAnalysisCommandOptions task(Function<Integer, Partition> task) {
+        this.task = requireNonNull(task);
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions task(Partition task) {
+        requireNonNull(task);
+        this.task = i -> task;
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions resultExtension(String extensionName) {
+        this.resultExtensions.add(requireNonNull(extensionName));
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions resultExtensions(Collection<String> extensionNames) {
+        this.resultExtensions.addAll(requireNonNull(extensionNames));
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions violationType(LimitViolationType violationType) {
+        this.violationTypes.add(requireNonNull(violationType));
+        return this;
+    }
+
+    public SecurityAnalysisCommandOptions violationTypes(Collection<LimitViolationType> violationTypes) {
+        this.violationTypes.addAll(requireNonNull(violationTypes));
+        return this;
+    }
+
+    private String pathToString(Path path) {
+        return (absolutePaths ? path.toAbsolutePath() : path).toString();
+    }
+
+    public SimpleCommand toCommand() {
+        Objects.requireNonNull(caseFile, "Case file is not defined.");
+        Preconditions.checkArgument(task == null || taskCount == null,
+                "Options task and task-count may not be defined together.");
+
+        SimpleCommandBuilder commandBuilder = new SimpleCommandBuilder()
+                .id(id)
+                .program(itoolsCommand != null ? itoolsCommand : "itools")
```

Comment:
The default itoolsCommand require that itools is in the path. It also consider that the OS is under Linux (or MacOS).